core/domain/question_fetchers_test.py

```diff
@@ -114,16 +131,19 @@ def test_cannot_get_question_from_model_with_invalid_schema_version(self):
         all_question_models = question_models.QuestionModel.get_all()
         self.assertEqual(all_question_models.count(), 0)
 
-        question_id = question_services.get_new_question_id()
+        question_id = question_services.get_new_question_id() # type: ignore[no-untyped-call]
 
-        question_model = question_models.QuestionModel(
+        question_model: Optional[
+            question_models.QuestionModel] = question_models.QuestionModel(
             id=question_id,
             question_state_data=(
-                self._create_valid_question_data('ABC').to_dict()),
+                self._create_valid_question_data('ABC').to_dict()), # type: ignore[no-untyped-call]
             language_code='en',
             version=0,
             question_state_data_schema_version=0)
 
+        # Ruling out the possibility of None for mypy type checking.
+        assert isinstance(question_model, question_models.QuestionModel)
```

Comment:
I think this is not needed since `question_model` is not `Optional`

Reply:
Done